src/Game/Replay.java

```diff
@@ -1163,6 +1173,16 @@ public static void dumpRawOutputStream(byte[] b, int off, int len) {
       output_checksum.update(buffer.array());
       output.write(buffer.array());
       output.flush();
+
+      /*
+      // Debug viewing entire output stream
+      System.out.print("Writing Output Stream: ");
```

Comment:
idem here, is it intended to have this for debug purposes or something?

Reply:
Yea, I actually specifically put it in in its own commit. I have had to uncomment it somewhat frequencly while debugging ISAAC sync.

Reply:
I'm figuring I may have to come back to it some day, or might be useful to uncomment.

Reply:
alright then